.ci/create_pkg.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
```

Comment:
Use bash -Ee to prevent unexpected behavior and error control; also, you can use the trap function.

Reply:
Will be added in next stage as improvments